pennylane/shadows/transforms.py

```diff
@@ -210,10 +210,5 @@ def circuit(x):
             [ 0.33705002-0.01125j   ,  0.011025  +0.232575j  ,
              -0.006075  +0.225225j  ,  0.33475   +0.j        ]],
            dtype=complex64, requires_grad=True)
-    >>> qml.jacobian(circuit)(x)
```

Comment:
Impossible to run this part with the jacobian, @eddddddy @Qottmann

Reply:
This line should be `qml.jacobian(lambda x: np.real(circuit(x)))(x)`, since autograd doesn't support complex gradients. 

You can keep the output below this. I think I just copied the previous line wrong.